fairer-iOS/fairer-iOS/Network/DTO/WorkInfoReponse.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HouseWorksResponse.swift
+//  fairer-iOS
+//
+//  Created by 홍준혁 on 2023/03/05.
+//
+
+import Foundation
+
+typealias WorkInfoReponse = [String: DayHouseWorks]
+
+struct DayHouseWorks: Codable {
+    let memberId: Int
+    let scheduledDate: String
+    let countDone, countLeft: Int
```

Comment:
p3;
```suggestion
    let countDone: Int
    let countLeft: Int
```
가독성을 위해서 분리해주시는 건 어떨까요??

Reply:
[afdcb22](https://github.com/fairer-iOS/fairer-iOS/pull/113/commits/afdcb22fdc9a486f33c35df28e1901fc94df5f91)